public/docs/ts/latest/guide/security.jade

```diff
@@ -1,115 +1,113 @@
 block includes
   include ../_util-fns
 :marked
-  Web application security has many aspects. This chapter describes Angular's built in
-  protections against common web application vulnerabilities and attacks, such as Cross Site
-  Scripting Attacks. It does not cover application level security, such as authentication (_Who is
+  This section describes Angular's built-in
+  protections against common web application vulnerabilities and attacks such as Cross-Site
+  Scripting Attacks. It does not cover application-level security, such as authentication (_Who is
   this user?_) or authorization (_What can this user do?_).
 
-  The [Open Web Application Security Project (OWASP)](https://www.owasp.org/index.php/Category:OWASP_Guide_Project)
-  has further information on the attacks and mitigations described below.
+  For more information about the attacks and mitigations described below, see [Open Web Application Security Project (OWASP)](https://www.owasp.org/index.php/Category:OWASP_Guide_Project).
```

Comment:
I'm not sure about the title (or destination) of this link. Maybe change to:

see the [OWASP Guide Project](...).